packages/dialect-react-ui/components/NotificationsButton/index.tsx

```diff
@@ -103,6 +135,14 @@ function WrappedNotificationsButton(
         colors.primary
       )}
     >
+      {hasNewMessages && (
+        <div
+          className={cs(
```

Comment:
Minor: `cs` is our custom function that we've started replacing with `clsx` package. Could you please update to `clsx` instead?